### ISSUE TITLE
Accessibility - Landing Page - color contrast issue with "More on BuiltWithDot.Net" and other hyperlinks

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#296]
Page: https://dotnetfoundation.org/

The "More on BuiltWithDot.Net>" link has an insufficient color contrast ratio of 3.97. WCAG 2 level AA requires a contrast ratio of 4.5:1 or greater for normal text.

Screenshot showing Accessibility Insights for Web failures:
![image](https://user-images.githubusercontent.com/4643903/89970586-22913b80-dc27-11ea-8a6c-123fa36bf9b2.png)

Full failure details:

**Title**: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="builtwithdot\.net"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

**Issue**: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

**Target application**: .NET Foundation - https://dotnetfoundation.org/

**Element path**: a[href$="builtwithdot\.net"]

**Snippet**: <a href="https://builtwithdot.net">More on BuiltWithDot.Net&gt;</a>

**How to fix**: 
Fix any of the following:

-  Element has insufficient color contrast of 3.97 (foreground color: #007bff, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1

**Environment**: Chrome version 84.0.4147.105

====

This accessibility issue was found using Accessibility Insights for Web 2.20.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by terrimorton** [comment]
This issue happens on all links across the site with a smaller font size.

My suggestion here (if you are not able to get a designer's input) is to change #007bff to #0051FF sitewide for link colors:
![image](https://user-images.githubusercontent.com/4643903/90052221-d1c12780-dca6-11ea-81a4-3f8afb5e0334.png)


Before (#007bff):
![image](https://user-images.githubusercontent.com/4643903/90052038-945c9a00-dca6-11ea-87c3-afa6a79c353e.png)


After (#0051FF):
![image](https://user-images.githubusercontent.com/4643903/90052122-aa6a5a80-dca6-11ea-8f11-9c33b257389e.png)

**Comment by terrimorton** [comment]
Also for reference, this is what the top part of https://dotnetfoundation.org/blog would look like using #006CFF for links:

![blog mockup - 2020-08-14_14-46-22](https://user-images.githubusercontent.com/4643903/90283049-75444080-de3d-11ea-946a-27b4b2a12c61.jpg)

**Comment by Arhell** [comment]
@terrimorton done
Thank you for accessibility research

**Comment by terrimorton** [comment]
I can see these updates on the website and the contrast issues for links are resolved. Thanks, @Arhell!
